cmd/hamctl/main.go

```diff
@@ -28,7 +28,6 @@ func main() {
 	c.AddCommand(versionCmd)
 	err = c.Execute()
 	if err != nil {
-		fmt.Printf("Error: %v\n", err)
```

Comment:
Cobra already logs errors from `Execute` so this is removed to avoid printing the error and reference twice.